libbindgen/tests/headers/dash_language.h

```diff
@@ -0,0 +1,6 @@
+// bindgen-flags: -- -x c++ --std=c++11
```

Comment:
This is related because before that, that gives an error due to us sending the header before `-x c++`, and treats it as C because it's a `.h`.